.github/workflows/ci.yml

```
@@ -57,9 +57,9 @@ jobs:
       - name: Install frontend dependencies
         working-directory: tsv-tennis-app
         run: npm ci
-      - name: Run ESLint
+      - name: Run Oxlint
         working-directory: tsv-tennis-app
-        run: npm run lint --if-present
+         run: npm run lint
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)